### ISSUE TITLE
Is it AWS EKS ready?

### ISSUE THREAD

**Issue by frederiksf** [weaveworks/scope#3542]
It it EKS ready? Did anyone use it successfully?

**Comment by bboreham** [comment]
Yes, Scope works with EKS. There are no EKS-specific features, but only standard APIs are used.
